.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+## Contributing
+
+First, thanks for considering contributing to our project! In general, our
+`Development.md` contains a lot of development-related information that could
+prove useful to a contributor!
+
+### Issues
+
+Before creating a new issue, look at current and past issues both on Github
+**AND** MongoDB JIRA. Also look at outstanding PRs, and overall, ensure that
+the issue hasn't already been reported. In particular, MongoDB's JIRA has, at
+the time of writing this, most of the active development information on this
```

Comment:
let's remove "at the time of writing this", I think it will always be true.

Reply:
Done.